server/backend/mlx_engine/model_kit/vision_add_ons/mistral3.py

```
@@ -0,0 +1,122 @@
+        prompt_tokens: mx.array,
+        images_b64: list[str],
+        max_size: tuple[int, int] | None,
+    ) -> tuple[mx.array, mx.array]:
+        """
+        Compute embeddings for text with images.
+
+        This method is heavily based on the mlx-vlm's mistral3 `get_input_embeddings`
+        https://github.com/Blaizzy/mlx-vlm/blob/2c3014fd40962bd5320ad611502e7e26cae08926/mlx_vlm/models/mistral3/mistral3.py#L240-L279
+        """
+
+        input_ids, pixel_values, attention_mask, other_model_inputs = (
```

Ruff (0.14.11)

Unpacked variable `attention_mask` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)